main/getting-started/intro-hackathon.md

```diff
@@ -0,0 +1,103 @@
+  of what they put up for it. 
+3. **[Agoric Platform/Stack](/platform/README.md)**: A brief introduction to the complete Agoric platform/technical stack.
+4. **[Agoric Dapp Guide](/dapps/README.md)**: 
+   A Dapp is a distributed application, typically with a browser-based user interface, a public
+   API server, and a contract running on the Agoric blockchain. This document explains a Dapp's
+   basic directory and file structure.
+
+There are several ways you can go from here:
+- Watch videos showing how to write smart contracts on the Agoric platform.
+- Read the more detailed Guides and look at our APIs.
+- Look at our already written sample contracts. 
+
+For general help, you should be aware of and make use of:
+- **[Agoric Glossary](/glossary/)**: Our Glossary focuses on terms we've given Agoric-context
+  definitions to (i.e. what does *mint* mean in an Agoric context?).
+
+## Videos
+
+We have two videos of talks given to external audiences that are particularly useful to contract developers.
+
+- [DeFi Workshops: Smart Contracts in JavaScript](https://www.youtube.com/watch?v=qudVWjSqDJU)
+  by Dean Tribble covers general concepts with examples.
+  
+- [Building a Composable DeFi Contract](https://www.youtube.com/watch?v=faxrecQgEio) by Kate Sills walks through programming first a covered call option contract,
+and then writing an OTC Desk contract that uses the covered call contract.
+
+## ERTP and Zoe Guides and APIs
+
+For a deeper explanation of Agoric concepts, what you can accomplish with a contract,
+and what Agoric API commands do, read the following:
+
+- ERTP: The **[ERTP Guide](/ertp/guide/README.md)** is a detailed description of 
+  ERTP concepts, design, components, and commands. The ERTP API is documented
+  [here](/ertp/api/#ertp-api). 
+- Zoe: The **[Zoe Guide](/zoe/guide/README.md)** is a detailed description of 
+  Zoe concepts, design, components, and commands. The Zoe API is documented
+  [here](/zoe/api/#zoe-api).
+  
+## Example contracts
+
+To familiarize yourself with working Agoric smart contracts, take a look at our 
+[sample contracts](/zoe/guide/contracts/README.md). They are:
+- [Atomic Swap](/zoe/guide/contracts/atomic-swap.md)
+- [Autoswap](/zoe/guide/contracts/autoswap.md)
+- [Barter Exchange](/zoe/guide/contracts/barter-exchange.md)
+- [Covered Call](/zoe/guide/contracts/covered-call.md)
+- [Multipool Autoswap](/zoe/guide/contracts/multipoolAutoswap.md)
+- [Second-price auction](/zoe/guide/contracts/second-price-auction.md)
+- [Simple Exchange](/zoe/guide/contracts/simple-exchange.md)
+
+## Advanced or specialized documentation
+
+As you start writing contracts, you should read our
+**[JavaScript Distributed Programming Guide](/distributed-programming.md)**. 
+We've made some Agoric-specific additions at various layers, including concepts, syntax, 
+and additions to the Agoric library. You should know about and understand these before 
+doing significant programming on the Agoric platform.
```

Comment:
If they are Agoric-specific additions, they must have been additions to things the developer might already have been familiar with. I suspect you mean javascript, blockchain, and perhaps distributed programming, but this doesn't say any of that.

Here's a start at actually saying that.

```suggestion
**[JavaScript Distributed Programming Guide](/distributed-programming.md)**. 
The Agoric platform allows you to write smart contracts in JavaScript, and run them where they'll be able to communicate with other programs running on Cosmos, Ethereum, or BitCoin. We've made some Agoric-specific additions to JavaScript, and added functionality that makes distributed programming both more secure and more approachable than other platforms. This guide presents concepts, syntax, 
and tools available on our platform. You should know about and understand these before 
doing significant programming on the Agoric platform.
```